vhdbuilder/packer/install-dependencies.sh

```diff
@@ -265,6 +266,8 @@ while IFS= read -r p; do
           installStandaloneContainerd "${version}"
         fi
         echo "  - containerd version ${version}" >> ${VHD_LOGS_FILEPATH}
+        # k8s will use images in the k8s.io namespaces - create it
+        ctr namespace create k8s.io
```

Comment:
Do we still need the `ctr` command here? Thought it's all being replaced with `crictl`?

Reply:
crictl is just calling ctr under the hood I believe, so I do think we still need this

Reply:
I wasn't sure if crictl was default to the namespace K8s.io. so I kept it to double check if they would land in K8s.oi still

Reply:
removed